worklets/Overview.bs

```diff
@@ -265,24 +267,26 @@ When a user agent is to <dfn>set up a worklet environment settings object</dfn>,
 
         : The <a>responsible document</a>
         :: Not applicable (the <a>responsible event loop</a> is not a <a>browsing context</a>
-                <a>event loop</a>).
+            <a>event loop</a>).
 
         : The <a>API URL character encoding</a>
-        :: Return UTF-8.
+        :: Return <a>UTF-8</a>.
 
         : The <a>API base URL</a>
         :: Return |inheritedAPIBaseURL|.
 
-        : The <a>origin</a> and <a>effective script origin</a>
+        : The <a>origin</a>
         :: Return |inheritedOrigin|.
```

Comment:
Is there any reason to be same-origin with your creator? Might be safer to return an opaque origin. Though I guess that should be considered separately.